setup.py

```diff
@@ -0,0 +1,20 @@
+from setuptools import setup
+
+import bus_sdk
+
+setup(
+    name='bus_sdk',
+    version=bus_sdk.__version__,
+    description='SDK for the eLife Sciences bus',
```

Comment:
link to https://github.com/elifesciences/bus somewhere wold be helpful

Reply:
Done.